hybridse/include/codec/list_iterator_codec.h

```diff
@@ -94,9 +94,7 @@ class ColumnImpl : public WrapListImpl<V, Row> {
 
     // TODO(xxx): iterator of nullable V
     std::unique_ptr<ConstIterator<uint64_t, V>> GetIterator() override {
-        auto iter = std::unique_ptr<ConstIterator<uint64_t, V>>(
-            new ColumnIterator<V>(root_, this));
-        return std::move(iter);
+        return std::make_unique<ConstIterator<uint64_t, V>>(root_, this);
```

Comment:
guess that should be `make_unique<ColumnIterator>`

Reply:
This `std::make_unique<ColumnIterator<V>>(root_, this);` is what I think the return statement should be, although I wasn't sure if I were to test the code mentioned in the issue or if I were to correct it. I'll try to force push that, let me know if there's anything I missed/should change.

Reply:
you are correct, `ColumnIterator` is sub-class of `Constiterator`